dev/devicelab/manifest.yaml

```diff
@@ -458,12 +458,12 @@ tasks:
     stage: devicelab_ios
     required_agent_capabilities: ["mac/ios32"]
 
-  flutter_gallery_ios32__transition_perf:
-    description: >
-      Measures the performance of screen transitions in Flutter Gallery on
-      32-bit iOS (iPhone 4S).
-    stage: devicelab_ios
-    required_agent_capabilities: ["mac/ios32"]
+  # flutter_gallery_ios32__transition_perf:
```

Comment:
Can this just be removed instead of commented out please? Also, please file a bug about the now deleted test with relevant details. I'll make a note in the sheriff playbook requesting guidance on how to handle scenarios like this in the future.

Reply:
At first I deleted everything but saw that in the file elsewhere people were just commenting them out.  If you want me to delete it do you want me to delete the test as well?

Reply:
> At first I deleted everything but saw that in the file elsewhere people were just commenting them out.

Oh. I didn't realize that was common here. Not a fan of commented out or dead code. So I'd always suggest deleting the code here and the test as well. We can track the intent to replace the test in a separate bug and the code will always be in git.

Reply:
done